### DISCUSSION TITLE
Problem : login Swift Code Page Application : login.swift

### DISCUSSION THREAD

**Discussion by dimitrihamelin** [community/community#79307] [Other Feature Feedback, Questions, & Ideas]
### IOS bug Login Page 

Question

### Body

Hello, I'm facing an issue in my code: when a user taps on something (just password), it seems to freeze the keyboard. Could someone please check my code. 

Necessary : Swift Code, IOS 17.

Exp : First App 

Link : https://github.com/dimitrihamelin/application

The problem is on the login.swift

**Comment by alcere** [comment]
Hi @dimitrihamelin!

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project.
It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by dimitrihamelin** [reply]
Hello thank you so much for your message. I have GitHub a few month. I'm so sorry, 
Do you think that I need to erase this discussion because I need some help to resolve this IOS bug, 
Best regards, 
Dimitri
